generators/wp/templates/wordpress-alpine/tests/bootstrap.php

```diff
@@ -1,36 +1,16 @@
 <?php
-define( 'IS_UNIT_TESTING', true );
-
-// Load Core's test suite.
-$_tests_dir = getenv( 'WP_TESTS_CONFIG_FILE_PATH' );
-require_once $_tests_dir . '/includes/functions.php';
-
 /**
- * Setup our environment (theme, plugins).
+ * IMPORTANT: Do not add any code before this require_once statement below
  */
-tests_add_filter( 'muplugins_loaded', function () {
-
-	error_reporting( E_CORE_ERROR | E_CORE_WARNING | E_COMPILE_ERROR | E_ERROR | E_WARNING | E_PARSE | E_USER_ERROR | E_USER_WARNING | E_RECOVERABLE_ERROR ); //@codingStandardsIgnoreLine
-
-	if ( ! defined( 'JETPACK_DEV_DEBUG' ) ) {
-		define( 'JETPACK_DEV_DEBUG', true );
-	}
-
-
-	// workaround falt errors where $wp_rewrite object have not been initialized.
-	remove_action( 'switch_theme', 'rri_wpcom_action_switch_theme' );
-
-	switch_theme( getenv( 'WORDPRESS_THEME' ) );
-
-	// load mocks - this should load after any plugins that should be loaded
-	require_once __DIR__ . '/helpers/plugin-function-mocks.php';
-
-} );
+require_once getenv('PMC_PHPUNIT_BOOTSTRAP');
 
-tests_add_filter( 'after_setup_theme', function () {
-	pmc_load_plugin( 'pmc-unit-test', 'pmc-plugins' );
+tests_add_filter('after_setup_theme', function () {
+	// @TODO: register additional namespaces, load plugins, etc...
 
-} );
+});
 
-// Include core's bootstrap.
-require $_tests_dir . '/includes/bootstrap.php';
+PMC\Unit_Test\Bootstrap::get_instance()->start([
+	'theme'      => getenv( 'WORDPRESS_THEME' ),
+	'namespace'  => getenv( 'WORDPRESS_THEME_NAMESPACE' ),
```

Comment:
@hauvong I'm curious how can we best utilize this in testing?

Reply:
We should remove thi, and move the code into common pmc bootstrap, most likely we only need PMC\Unit_Test\Bootstrap::get_instance()->start().  Bootstrap already have some of the value auto detect.   We only need this on rare cause where them might not follow standard or define those env value properly.  I expose this as an option while I was working with bootstrap, we can do most of this in boostrap already for auto detection.